scripts/convert_llama_to_dcp.py

```diff
@@ -0,0 +1,63 @@
+import argparse
+import json
+from pathlib import Path
+
+import torch
+import torch.distributed.checkpoint as DCP
+
```

Comment:
Let's do linting to the code. See https://github.com/pytorch/torchtitan/blob/main/CONTRIBUTING.md?plain=1#L17